workflows/nallo.nf

```diff
@@ -646,7 +647,8 @@ workflow NALLO {
             FILTER_VARIANTS_SVS (
                 ch_clin_research_svs_vcf.clinical,
                 ch_hgnc_ids,
-                params.filter_variants_hgnc_ids
+                params.filter_svs_expression,
+                params.filter_variants_hgnc_ids,
```

Comment:
Same here, does this also need the boolean flags here for the two filter options?